diff --git a/node_modules/@coreui/coreui/dist/js/coreui.esm.js b/node_modules/@coreui/coreui/dist/js/coreui.esm.js
index bde1f4d..3ea50f4 100644
--- a/node_modules/@coreui/coreui/dist/js/coreui.esm.js
+++ b/node_modules/@coreui/coreui/dist/js/coreui.esm.js
@@ -4287,7 +4287,9 @@ var Tooltip = /*#__PURE__*/function () {
 
       EventHandler.trigger(_this2.element, _this2.constructor.Event.HIDDEN);
 
-      _this2._popper.destroy();
+      if(_this2._popper) {
+        _this2._popper.destroy()
+      };
     };
 
     var hideEvent = EventHandler.trigger(this.element, this.constructor.Event.HIDE);
@@ -5705,15 +5707,17 @@ var Sidebar = /*#__PURE__*/function () {
         Default$9.activeLinksExact = JSON.parse(dataAttributes.activeLinksExact);
       }
 
+      let definitySame=false;
       if (Default$9.activeLinksExact && element.href === currentUrl) {
         element.classList.add(CLASS_NAME_ACTIVE$4); // eslint-disable-next-line unicorn/prefer-spread
 
         Array.from(_this4._getParents(element, SELECTOR_NAV_DROPDOWN$1)).forEach(function (element) {
           element.classList.add(CLASS_NAME_SHOW$6);
         });
+        definitySame=true;
       }
 
-      if (!Default$9.activeLinksExact && element.href.startsWith(currentUrl)) {
+      if (!definitySame && Default$9.activeLinksExact && element.href.startsWith(currentUrl)) {
         element.classList.add(CLASS_NAME_ACTIVE$4); // eslint-disable-next-line unicorn/prefer-spread
 
         Array.from(_this4._getParents(element, SELECTOR_NAV_DROPDOWN$1)).forEach(function (element) {
